src/psyclone/tests/domain/lfric/lfric_loop_test.py

```diff
@@ -261,39 +260,23 @@ def test_lower_to_language_normal_loop():
     assert loop1.start_expr.symbol.name == "loop1_start"
 
 
-def test_lower_to_language_domain_loop():
-    ''' Tests that we can call lower_to_language_level on a domain LFRicLoop.
-    This test takes an invoke with two consecutive domain kernels and then
-    fuses the 'loops' to verify that the kernels are all still in the right
-    order.
+def test_lower_to_language_null_loop():
+    ''' Tests that we can call lower_to_language_level on a NULL LFRicLoop.
```

Comment:
Strictly speaking it is called a 'domain' LFRicLoop so I suggest "'domain'/NULL"

Reply:
Done